u_P: 1.5
u_I: 0.0
u_D: 1.0

v_P: 1.5
v_I: 0.0
v_D: 1.0

w_P: 1.5
w_I: 0.0
w_D: 1.0

x_P: 1.0
x_I: 0.0
x_D: 1.0

y_P: 1.0
y_I: 0.0
y_D: 1.0

z_P: 1.0
z_I: 0.0
z_D: 1.0

psi_P: 1.0
psi_I: 0.0
psi_D: 0.0

tau: 0.05

equilibrium_thrust: 0.68

max_roll: 0.15
max_pitch: 0.15
max_yaw_rate: 0.785
max_throttle: 1.0
max_u: 1.0
max_v: 1.0
